hbase-it/src/test/java/org/apache/hadoop/hbase/RESTApiClusterManager.java

```diff
@@ -334,7 +334,7 @@ public String toString() {
     roleServiceType.put(ServiceType.HBASE_REGIONSERVER, Service.HBASE);
   }
 
-  private enum Service {
+  enum Service {
     HBASE, HDFS, MAPREDUCE
   }
 }
```

Comment:
oh. it never worked?

Reply:
I suspect it worked before the switch to GSON.